jest.config.js

```diff
@@ -69,7 +69,13 @@ module.exports = {
   // ],
 
   // A map from regular expressions to module names that allow to stub out resources with a single module
-  // moduleNameMapper: {},
+  moduleNameMapper: {
+    // {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/__mocks__/fileMock.js',
```

Comment:
After running the test suite, I find the file `styleMock.js`, but not `fileMock.js`

Am I missing something? What's the intent behind this config setting?

Reply:
Thats my bad. I should have deleted the fileMock segment.